src/Keboola/DbExtractor/Test/ExtractorTest.php

```diff
@@ -74,21 +85,21 @@ protected function getEnv(string $driver, string $suffix, bool $required = false
     {
         $env = strtoupper($driver) . '_' . $suffix;
         if ($required) {
-            if (false === getenv($env)) {
-                throw new \Exception($env . " environment variable must be set.");
+            if (false === (bool) getenv($env)) {
```

Comment:
no tak premyslim, proc to vlastne nemuze byt:

```suggestion
            if (getenv($env)) {
```

Reply:
to by asi mohlo, ale s "!"

Reply:
jezis ja jsem pako :), jj s !